src/plugins/manus/install-dependencies.sh

```
@@ -0,0 +1,161 @@
+if [ ! -f "/etc/ld.so.conf.d/local-libs.conf" ]; then
+    echo "/usr/local/lib" | tee /etc/ld.so.conf.d/local-libs.conf > /dev/null
+    ldconfig
+    echo "Added /usr/local/lib to ld.so.conf"
+fi
+
+# Add read/write permissions for manus devices
+if [ ! -f "/etc/udev/rules.d/70-manus-hid.rules" ]; then
+    mkdir -p /etc/udev/rules.d/
+    touch /etc/udev/rules.d/70-manus-hid.rules
+    echo "[Setup] Adding read/write permissions for manus devices..."
+    echo "# HIDAPI/libusb" >> /etc/udev/rules.d/70-manus-hid.rules
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)